### ISSUE TITLE
Make BoxHelper workable with Object3D

### ISSUE THREAD

**Issue by gdauster** [mrdoob/three.js#6837]
I added this simple peace of code in BoxHelper (at the top of the update function) for Object3D that are Mesh container, so they don't have geometry property. 

```javascript	
var min, max;
if (object.type == 'Object3D') {
	var box = new THREE.Box3().setFromObject(object);
	min = box.min;
	max = box.max;
	} else if (object.type == 'Mesh') {
		var geometry = object.geometry;
		if ( geometry.boundingBox === null ) {
			geometry.computeBoundingBox();
		}
		min = geometry.boundingBox.min;
		max = geometry.boundingBox.max;
		this.matrix = object.matrixWorld;
	}
}
```

Maybe this can be added to the offical source ?

**Comment by WestLangley** [comment]
The suggested enhancement is to get `BoxHelper` to work with hierarchical objects --- as `Box3` does.

**Comment by mrdoob** [comment]
Sounds good to me.

**Comment by gero3** [comment]
Fixed by #6837
